samples/qa-voice-control/README.md

```diff
@@ -0,0 +1,9 @@
+## Enact Component patterns for voice control features
```

Comment:
please remove this entire sample app and it's corresponding testcases
We don't have voice control support in Agate